XenModel/Messages.resx

```diff
@@ -13839,6 +13839,11 @@ Schedule:
   <data name="WARNING_HEALTH_CHECK_SERVICE_INFO" xml:space="preserve">
     <value>The Health Check service has been removed in {0}. If you are going to upgrade the pool to this version, you must disable Health Check first.</value>
   </data>
+  <data name="WARNING_PRE_CLOUD_VERSION_CONNECTION" xml:space="preserve">

```

Comment:
I think we can make this message a bit shorter and be specific about the version since we disallow connection to others:
`<value>{0} is not recommended for general management of servers running {1} {2}. Please use {3} {2} or upgrade {1} using the Rolling Pool Upgrade wizard.</value>`
where the placeholders are BrandManager's `BrandConsole`, `ProductBrand`, `ProductVersion82`, `LegacyConsole` (this needs to be added)

Reply:
Fixed.